DEFEDIT-547 Improve error dialog

ragnard-king opened the pull request:
- new dialog for unhandled exceptions with ability to report as github issue
  - includes link to sentry issue, if available
- always log
- suppress sentry reporting and user notification of exceptions that repeat with less than 1s time in between

Fixes: https://github.com/defold/editor2-issues/issues/281

ragnard-king committed:
DEFEDIT-547 Improve error dialog

matgis-king reviewed (changes requested):
Looks nice overall! Some concerns might need addressing.

ragnard-king committed:
DEFEDIT-547 Review fixes

- fixed incorrect link to sentry
- removed redundant code
- removed unused scss variable
- better exception message
  - include type
  - default to "Unknown" if message is null

ragnard-king committed:
Merge branch 'dev' into DEFEDIT-547-improve-error-dialog

ragnard-king committed:
DEFEDIT-547 Fix aot compilation error

- schema.utils is now required in a different order than before which
  broke things, exclude from aot like we do with schema.core
